src/NewTools-FileBrowser/StFileDialogPresenter.class.st

```diff
@@ -320,7 +320,9 @@ StFileDialogPresenter >> open [
 StFileDialogPresenter >> openModal [
 
 	super openModal cancelled ifTrue: [ ^ nil ].
-	^ self selectedEntry
+	self isMultipleSelection 
```

Comment:
I was wondering if we should not use the selection.
Then I do not understand because we should not have a method returning a collection or an element. 
My impression is that we should use selectedEntries because it should return either a list with a single or multiple elements.

Reply:
I didn't want to change it now because it could break other tools, but I will check because not many tools are using this now.